nekoyume/Assets/_Scripts/State/LocalLayerModifier.cs

```diff
@@ -290,16 +274,13 @@ long blockIndex
                 return;
             }
 
-            TryResetLoadedAvatarState(
-                avatarAddress,
-                out var outAvatarState,
-                out var isCurrentAvatarState);
+            TryResetLoadedAvatarState(avatarAddress, out _, out _);
         }
 
         public static void RemoveAttachmentResult(
             Address avatarAddress,
             Guid mailId,
-            bool resetState = true)
+            bool resetState = default)
```

Comment:
I know the bool default value is false, is this okay?

Reply:
Yes it is. I have fixed all other arguments.